connman/src/main.c

```diff
@@ -544,14 +549,25 @@ static void parse_config(GKeyFile *config)
 		connman_settings.fallback_device_types =
 				parse_fallback_device_types(str_list, len);
 
+	g_strfreev(str_list);
+
 	g_clear_error(&error);
 
-	boolean = __connman_config_get_bool(config, "General",
+	boolean = __connman_config_get_bool(config, group,
 				CONF_ENABLE_LOGIN_MANAGER, &error);
 	if (!error)
 		connman_settings.enable_login_manager = boolean;
 
-	g_strfreev(str_list);
+	g_clear_error(&error);
+
+	str = __connman_config_get_string(config, group, CONF_LOCALTIME,
+				&error);
+	if (!error) {
+		connman_settings.localtime = str;
```

Comment:
This is not `free()`d on shutdown.

Reply:
This should take care of it 481c63fe79c452579491f83ab27753ac3f6d4224